lesson4/hometask_frameq.rb

```diff
@@ -0,0 +1,29 @@
+class LazyHash < Hash
+  def method_missing(name)
+    key = name.to_s
+    super.method_missing(key) unless self.key?(key)
```

Comment:
And what's happened if your hash would contain a _symbol_, not a _string_? I don't think that `self.key?(key)` would handle it.